### PR TITLE
Correct misnamed variable in Schema.php.

### PR DESCRIPTION
**Pull request by coderabbi**
Didn't think you were getting off that easy, did you? ;-)

### CODE REVIEW

**Comment by shadowhand** [comment]
Thanks!

**Comment by shadowhand** [comment]
Released in 0.4.0.

File: src/Schema.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public static function make($json): Schema
         // @codeCoverageIgnoreStart
         throw new \InvalidArgumentException(sprintf(
             "No schema type available for %s",
-            $schema->type
+            $json->type
```
Comment (shadowhand) [inline src/Schema.php:52]: FWIW, I'd love to have tests for this. It's a total 🏴 that this is coverage ignored.

**Comment by coderabbi** [comment]
Heh. I wondered about that. Let me see ambitious I feel in the morning,
sounds like a worthy pursuit. :-)

**Pull request merged**